packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -41,12 +41,6 @@ const EDiscovery = WebexPlugin.extend({
   namespace: 'EDiscovery',
 
   session: {
-    contentSummaryBySpaceIdCache: {
-      type: 'object',
-      default() {
-        return new Map();
-      }
-    },
```

Comment:
The `contentSummaryBySpaceIdCache` holds the results returned from ediscovery's `/summary` API. i.e. Metadata about each _space_ in a report. The cache gives the client faster performance and prevents rate limiting on the server. Previously when the eDiscovery Downloader started a report it would pre-populate this cache.

Later a new API was introduced as eDiscovery expanded to handle meetings as well as spaces. The `/container` API replaces `/summary` and has been adopted by the eDiscovery Downloader. However the old API remains on the server to support legacy clients.

But there was a dependency on `contentSummaryBySpaceIdCache` in `transforms.js` that was missed. When the cache was no longer pre-populated the code used a fallback strategy. It queried the `/summary` API for all spaces not found in the cache.

When the query by space id hits a network error it gives up. This meant all activities for that space cannot be decrypted. Essentially the space will be missing from the final eDiscovery report.

This is being fixed here by removing all references to, and uses of, the `contentSummaryBySpaceIdCache`. That includes the removal of the old `/summary` APIs. Instead the code will use the pre-populated `contentContainerCache`.